src/components/organisms/TriggersFormItems/ResourceTriggerSelect/ResourceTriggerSelect.tsx

```diff
@@ -0,0 +1,88 @@
+import {useContext, useMemo} from 'react';
+
+import {Form, Select} from 'antd';
+
+import {ReactComponent as TestSuitesIcon} from '@assets/test-suites-icon.svg';
+
+import {ExecutorIcon} from '@atoms';
+
+import {MainContext} from '@contexts';
+
+import {Text} from '@custom-antd';
+
+import {TestForTrigger} from '@models/test';
+import {TestSuiteForTrigger} from '@models/testSuite';
+
+import {useAppSelector} from '@redux/hooks';
+import {selectExecutors} from '@redux/reducers/executorsSlice';
+import {getTestExecutorIcon} from '@redux/utils/executorIcon';
+
+import {useGetAllTestSuitesQuery} from '@services/testSuites';
+import {useGetAllTestsQuery} from '@services/tests';
+
+import Colors from '@styles/Colors';
+
+import {required} from '@utils/form';
+
+import {StyledResourceOptionWrapper} from './ResourceTriggerSelect.styled';
+
+const {Option, OptGroup} = Select;
+
+const ResourceTriggerSelect = () => {
+  const {isClusterAvailable} = useContext(MainContext);
+
+  const executors = useAppSelector(selectExecutors);
+
+  const {data: tests = []} = useGetAllTestsQuery(null, {skip: !isClusterAvailable});
+  const {data: testSuites = []} = useGetAllTestSuitesQuery(null, {
+    skip: !isClusterAvailable,
+  });
+
+  const testsData: TestForTrigger[] = useMemo(() => {
```

Comment:
It should be resolved easily by `tests` type
```suggestion
  const testsData = useMemo(() => {
```